src/azure-cli/azure/cli/command_modules/appservice/_help.py

```diff
@@ -2293,12 +2293,12 @@
       text: az staticwebapp list
 """
 
-helps['staticwebapp browse'] = """
+helps['staticwebapp show'] = """
```

Comment:
changing command interface is breaking change to azure cli. There are several things to do:
1. Add BREAKING CHANGE tag in release note
2. The right behavior to rename an existing command is you add a new command and deprecate previous one according to https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#deprecating-commands-and-arguments